### PR TITLE
Added a link to broaden description on SSL

### PR DESCRIPTION
**Pull request by MatthewBellwood**

### CODE REVIEW

File: Reference/Security/index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ We have a dedicated security page on our main site which provides most of the de
 
 ## SSL/HTTPS
 
-We highly encourage the use of HTTPS on Umbraco websites especially in production environments. By using HTTPS you greatly improve the security of your website.
+We highly encourage the use of [SSL](https://hosting.review/web-hosting-glossary/#12)/HTTPS on Umbraco websites especially in production environments. By using HTTPS you greatly improve the security of your website.
```
Comment (jmayntzhusen) [inline Reference/Security/index.md]:
```suggestion
We highly encourage the use of [SSL](https://cheatsheetseries.owasp.org/cheatsheets/Transport_Layer_Protection_Cheat_Sheet.html#providing-transport-layer-protection-with-ssltls)/HTTPS on Umbraco websites especially in production environments. By using HTTPS you greatly improve the security of your website.
```

Comment (jmayntzhusen) [reply Reference/Security/index.md]: Hey @MatthewBellwood

Thanks a lot for making a contribution to our documentation. I agree that explaining a bit more why changing to SSL is a good idea would benefit the documentation.

We don't like linking to sites we don't know though, so instead I replaced your suggested link with OWASP's own description of why to use TLS 🙂

**Comment by jmayntzhusen** [comment]
As I haven't heard back from you and I see this is a brand new GH account that has added this specific link to a bunch of docs repos I will close this. 😕

**Comment by MatthewBellwood** [comment]
That's alright, I get it. Thanks, anyway.

**Pull request closed without merging**